graph/util.go

```diff
@@ -0,0 +1,38 @@
+package graph
+
+import "net/http"
+
+type Response struct {
+	Message string
+	Code    int
+}
+
+// Error panics with InternalServerError and the provided message
+func Error(message string) Response {
+	panic(Response{
+		Message: message,
+		Code:    http.StatusInternalServerError,
+	})
+}
+
+// Forbidden panics with Forbidden and the provided message
+func Forbidden(message string) Response {
+	panic(Response{
+		Message: message,
+		Code:    http.StatusForbidden,
+	})
+}
```

Comment:
not that it matters, but why don't Forbidden and Error both just delegate to Panic?

```
func Error(message string) Response { Panic(message, http.StatusInternalServerError) }
func Forbidden(message string) Response { Panic(message, http.StatusForbidden) }
```

Reply:
yeah, that was silly. I blame Monday.  I'll change it...